### DISCUSSION TITLE
Seen button or Not Interested

### DISCUSSION THREAD

**Discussion by TheoOliveira** [dailydotdev/daily#624] [Feature Requests]
I've check so many articles but in case I don't want to see them anymore a seen button to make them disappear and appear another one, maybe. I know we have hide but hide is not quite seen, right.

**Comment by idoshamun** [comment]
Hi @TheoOliveira,

Have you seen our hide feature?
Open the card's context menu and hit hide. It should do the work
<img width="391" alt="image" src="https://user-images.githubusercontent.com/1993245/178143290-dc8cf595-8665-4655-8684-124fffdb89de.png">
